tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -762,6 +769,29 @@ EOF
 	info "Create /etc/resolv.conf file in rootfs if not exist"
 	touch "$dns_file"
 
+    if [ -n "${TEE_PLATFORM}" ]; then
+		if [ "${TEE_PLATFORM}" == "es" ]; then
```

Comment:
for SEV, we should just use the platforms `SEV` and `SNP`. From an attestation standpoint there is no difference between SEV and SEV-ES.